src/Type.cc

```diff
@@ -1041,6 +1050,12 @@ const char* RecordType::AddFields(const type_decl_list& others,
 	return nullptr;
 	}
 
+void RecordType::AddField(unsigned int field, const TypeDecl* td)
```

Comment:
Yes, that's a good thought.  I picked up the name just because its caller is `AddFields`, and the assert was to make sure I hadn't screwed anything up, but I'm okay with removing it now.  I've made this change and it will be part of the next set of updates I push.